kymatio/scattering2d/frontend/keras_frontend.py

```diff
@@ -0,0 +1,33 @@
+from kymatio.frontend.keras_frontend import ScatteringKeras
+from kymatio.scattering2d.frontend.base_frontend import ScatteringBase2D
+
+from kymatio.tensorflow import Scattering2D as ScatteringTensorFlow2D
+
+from tensorflow.python.framework import tensor_shape
+
+
+class ScatteringKeras2D(ScatteringKeras, ScatteringBase2D):
+    def __init__(self, J, L=8, max_order=2, pre_pad=False):
+        ScatteringKeras.__init__(self)
+        ScatteringBase2D.__init__(self, J, None, L, max_order, pre_pad,
+                                  'array')
+
+    def build(self, input_shape):
+        shape = tuple(tensor_shape.TensorShape(input_shape).as_list()[-2:])
+        self.S = ScatteringTensorFlow2D(J=self.J, shape=shape,
+                                        L=self.L, max_order=self.max_order,
+                                        pre_pad=self.pre_pad)
+        ScatteringKeras.build(self, input_shape)
+        # NOTE: Do not call ScatteringBase2D.build here since that will
```

Comment:
maybe this should be in the doc of `build`

Reply:
I don't think we're documenting the `build` methods anywhere, are we?